protocol/x/clob/flags/flags.go

```diff
@@ -127,5 +138,11 @@ func GetClobFlagValuesFromOptions(
 		}
 	}
 
+	if option := appOpts.Get(MaxDeleveragingSubaccountsToIterate); option != nil {
+		if v, err := cast.ToUint32E(option); err == nil {
+			result.MaxDeleveragingSubaccountsToIterate = v
```

Comment:
Is it worth creating a low-priority M6 ticket to validate this? Slightly worried that when people are setting this they could accidentally set it to zero and make it impossible to deleverage subaccounts.

Reply:
https://linear.app/dydx/issue/DEC-2146